server/monitor.go

```diff
@@ -2985,6 +3014,11 @@ func (s *Server) healthz() *HealthStatus {
 		return health
 	}
 
+	// If JSServerOnly is true, then do not check further accounts, streams and consumers.
+	if opts.JSServerOnly {
+		return health
```

Comment:
This one would be much harder to test since you would need server to be current, but not consumers.

Reply:
I don't have enough knowledge base of code base to write this type of test.